charts/openfga/values.schema.json

```diff
@@ -313,6 +313,40 @@
         "allowEvaluating1_0Models": {
             "type": ["boolean", "null"],
             "description": "allow calling Check, Expand, ListObjects, Write Assertions with models that have 1.0 schema version"
+        },
+        "annotations": {
```

Comment:
I think there may have been a misunderstanding here.

If we added the `annotations` to the top-level `values.yaml` body, then we only need an entry in the `values.schema.json` file for that single key/value pair. We don't need to enumerage all of the service, deployment, etc.. manifestations those annotations apply to.

We can remove the nested definitions for for `service.annotations` and the others. All we need is the `annotations` entry here.

Reply:
@jon-whit let know what you think based on my reply above, and based on it I'll refactor the PR to match the decision :)

Reply:
@PedroArvela I'll take a look today. Sorry for the delay.